ci-operator/jobs/coreos/coreos-assembler/coreos-coreos-assembler-rhel-9.6-presubmits.yaml

```
@@ -1,20 +1,21 @@
 presubmits:
-  opendatahub-io/openvino_model_server:
+  coreos/coreos-assembler:
   - agent: kubernetes
-    always_run: true
+    always_run: false
     branches:
-    - ^2025\.4-release$
-    - ^2025\.4-release-
-    cluster: build01
+    - ^rhel-9\.6$
+    - rhel-9\.6-
```

we lost the `^` here

```suggestion
    - ^rhel-9\.6-
```